ballista/rust/core/src/serde/logical_plan/to_proto.rs

```diff
@@ -777,9 +778,9 @@ impl TryInto<protobuf::LogicalPlanNode> for &LogicalPlan {
                     )))
                 }
             }
-            LogicalPlan::Projection {
+            LogicalPlan::Projection(ProjectionPlan {
```

Comment:
I think we can use the style, `LogicalPlan::Projection(p) => {p.var1, p.var2, ...}`.

FYI, https://github.com/apache/arrow-datafusion/issues/1228#issuecomment-969443381